uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/calendar-feeds/link-modal/calendar-feeds-link-modal.component.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash'
+], function (_) {
+  return {
+    __name: 'calendarFeedsLinkModal',
+    bindings: {
+      dismiss: '<',
+      url: '<'
+    },
+    controller: _.noop,
```

Comment:
do we need that? `_.noop` returns `undefined` afaik.

Reply:
We do need _.noop, it returns a **function** that returns undefined. Without a controller, it can't pass the bindings to the view.